File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/25/25.py
# Q25: Print number pyramid (1, 232, 34543, 4567654).
# Input: An integer n
# Output: Number pyramid pattern

n = int(input())
for i in range(1, n + 1):
    row = " " * (n - i)
    for j in range(i):
        print(i + j, end="")
    for j in range(i - 2, -1, -1):
        print(i + j, end="")
    print()

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/25/25.go
// Q25: Print number pyramid (1, 232, 34543, 4567654).
// Input: An integer n
// Output: Number pyramid pattern

package main

import (
    "fmt"
    "strings"
)

func main() {
    var n int
    fmt.Scan(&n)
    for i := 1; i <= n; i++ {
        row := strings.Repeat(" ", n-i)
        for j := 0; j < i; j++ {
            row += fmt.Sprintf("%d", i+j)
        }
        for j := i - 2; j >= 0; j-- {
            row += fmt.Sprintf("%d", i+j)
        }
        fmt.Println(row)
    }
}
